frontend/packages/client/src/hooks/useLeaderBoard.js

```diff
@@ -2,28 +2,27 @@ import { useQuery } from 'react-query';
 import { useErrorHandlerContext } from '../contexts/ErrorHandler';
 import fetchLeaderBoard from 'api/leaderboard';
 
-export default function useLeaderBoard({ communityId = 0 } = {}) {
+export default function useLeaderBoard({ communityId = 0, addr = '' } = {}) {
   const { notifyError } = useErrorHandlerContext();
   const { isLoading, isError, data, error } = useQuery(
-    ['leaderboard'],
-    async () => fetchLeaderBoard(communityId)
+    ['leaderboard', addr],
+    async () => fetchLeaderBoard(communityId, addr)
   );
 
   if (isError) {
     notifyError(error);
   }
 
+  const leaderBoard = data?.data?.Users ?? [];
+  const currentUser = data?.data?.CurrentUser;
```

Comment:
can we make this `users` and `currentUsers` ?

Reply:
Done - for currentUser, there will only ever be one user since that is the user whose wallet is connected to CAST. There will be no currentUser if wallet is not connected.

Reply:
my bad, it would be `currentUser`

Reply:
Updated backend with json tag to return `users`/`currentUser`